### PR TITLE
dropbearkey: add page

### PR DESCRIPTION
**Pull request by johandegn**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known): Dropbear v2022.83**

### CODE REVIEW

File: pages/linux/dropbearkey.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`dropbearkey -t {{ecdsa}} -f {{path/to/key}}`
+
+- Generate a SSH key of [t]ype RSA with 4096-bit key [s]ize and write it to key [f]ile:
+
+`dropbearkey -t {{rsa}} -s {{4096}} -f {{path/to/key}}`
+
+- Print the private key fingerprint and public key in key [f]ile:
+
+`dropbearkey -y -f {{pay/to/key}}`
```
Comment (kbdharun) [inline pages/linux/dropbearkey.md:20]:
```suggestion
`dropbearkey -y -f {{path/to/key_file}}`
```

File: pages/linux/dropbearkey.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Generate a SSH key of [t]ype ed25519 and write it to key [f]ile:
+
+`dropbearkey -t {{ed25519}} -f {{path/to/key}}`
+
+- Generate a SSH key of [t]ype ecdsa and write it to key [f]ile:
+
+`dropbearkey -t {{ecdsa}} -f {{path/to/key}}`
+
+- Generate a SSH key of [t]ype RSA with 4096-bit key [s]ize and write it to key [f]ile:
```
Comment (kbdharun) [inline pages/linux/dropbearkey.md:14]:
```suggestion
- Generate an SSH key of [t]ype RSA with 4096-bit key [s]ize and write it to key [f]ile:
```

File: pages/linux/dropbearkey.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`dropbearkey -t {{rsa}} -s {{4096}} -f {{path/to/key}}`
```
Comment (kbdharun) [inline pages/linux/dropbearkey.md:16]:
```suggestion
`dropbearkey -t {{rsa}} -s {{4096}} -f {{path/to/key_file}}`
```

File: pages/linux/dropbearkey.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> Generate SSH keys in Dropbear format.
+> More information: <https://manned.org/dropbearkey.1>.
+
+- Generate a SSH key of [t]ype ed25519 and write it to key [f]ile:
+
+`dropbearkey -t {{ed25519}} -f {{path/to/key}}`
+
+- Generate a SSH key of [t]ype ecdsa and write it to key [f]ile:
+
+`dropbearkey -t {{ecdsa}} -f {{path/to/key}}`
```
Comment (kbdharun) [inline pages/linux/dropbearkey.md:12]:
```suggestion
`dropbearkey -t {{ecdsa}} -f {{path/to/key_file}}`
```

File: pages/linux/dropbearkey.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# dropbearkey
+
+> Generate SSH keys in Dropbear format.
+> More information: <https://manned.org/dropbearkey.1>.
+
+- Generate a SSH key of [t]ype ed25519 and write it to key [f]ile:
+
+`dropbearkey -t {{ed25519}} -f {{path/to/key}}`
+
+- Generate a SSH key of [t]ype ecdsa and write it to key [f]ile:
```
Comment (kbdharun) [inline pages/linux/dropbearkey.md:10]:
```suggestion
- Generate an SSH key of [t]ype ecdsa and write it to key [f]ile:
```

File: pages/linux/dropbearkey.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# dropbearkey
+
+> Generate SSH keys in Dropbear format.
+> More information: <https://manned.org/dropbearkey.1>.
+
+- Generate a SSH key of [t]ype ed25519 and write it to key [f]ile:
+
+`dropbearkey -t {{ed25519}} -f {{path/to/key}}`
```
Comment (kbdharun) [inline pages/linux/dropbearkey.md:8]:
```suggestion
`dropbearkey -t {{ed25519}} -f {{path/to/key_file}}`
```

File: pages/linux/dropbearkey.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# dropbearkey
+
+> Generate SSH keys in Dropbear format.
+> More information: <https://manned.org/dropbearkey.1>.
+
+- Generate a SSH key of [t]ype ed25519 and write it to key [f]ile:
```
Comment (kbdharun) [inline pages/linux/dropbearkey.md:6]:
```suggestion
- Generate an SSH key of [t]ype ed25519 and write it to key [f]ile:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**